maven/bnd-maven-plugin/src/main/java/aQute/bnd/maven/plugin/BndMavenPlugin.java

```diff
@@ -89,6 +108,19 @@
     @Parameter(defaultValue = "false", readonly = true)
     private boolean				skip;
 
+    /* This exists only to allow documentation */
+	/**
+	 * The location of the bnd file that this plugin will supply to <pre>bnd</pre>.
+	 * The plugin combines the contents of bnd files from the project and its parents.
+	 * The plugin walks up the tree of parents. At each level, if the project as a <pre>basedir</pre>,
+	 * the plugin looks for a bnd file. If there is a local configuration of this parameter,
+	 * it determines the file name relative to that project's basedir; otherwise, it looks for the default,
+	 * <pre>${basedir}/bnd.bnd</pre>. For each project's bnd file, <pre>-include</pre> directives are resolved
+	 * relative to that project's basedir.
+	 */
+	@Parameter(defaultValue = "bnd.bnd")
+	String bndfile;
```

Comment:
This is wrong to put a field in that is unused to hang a doc comment on for a doc site we don't build or publish.